account_asset_management_xls/report/account_asset_report_xls.py

```diff
@@ -766,6 +766,21 @@ def _active_report(self, _p, _xs, data, objects, wb):
         row_pos = self.xls_write_row(
             ws, row_pos, row_data, row_style=self.rt_cell_style_right)
 
+    def render(self, wanted, col_specs, rowtype, render_space='empty'):
+        """
+        When date_remove is empty, default value `None` is assigned:
+        xls_types_default = {..., 'date': None, ...}
+        This causes following error:
+        `argument 2 must be datetime.time, not None`
+        To fix this case, set type to `text` and value to ''
+        """
+        row = super(AssetReportXls, self).render(
+            wanted, col_specs, rowtype, render_space=render_space)
+        if len(row) == 5 and row[3] == 'date' and row[4] == None:
```

Comment:
Best practice is to compare None using `is None` instead of `==`. It's slightly faster, and more predictable in the case of operator overloading.

Reply:
Thanks, @StefanRijnhart good to know.